### DISCUSSION TITLE
[Solved] blendhit.com

### DISCUSSION THREAD

**Discussion by redazul** [phantom/docs#192] [Help & Support] [answered]
### Summary

Hi https://blendhit.com is blocked and when I went to submit an issue ticket I see that a PR wasn't even done into the block list.

Either way not sure why it's blocked...it's just a standard CMv2 Minting landing page.
Help with this would be appreciated, Friday is suppose to be mint day.

### Example

https://blendhit.com

### Steps to Reproduce

Visit blendhit.com with phantom browser extension installed.

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
This should be resolved.
